codec-http2/src/test/java/io/netty/handler/codec/http2/Http2SettingsTest.java

```diff
@@ -75,4 +75,14 @@ public void settingsShouldSupportUnsignedShort() {
         settings.put(key, 123L);
         assertEquals(123L, (long) settings.get(key));
     }
+
+    @Test
+    public void boundarySettingsShouldBeSet() {
+        settings.maxHeaderListSize(Integer.MAX_VALUE + 1);
```

Comment:
I couldn't find the warning but it may be a mistake ... I favor the first idea because this PR contains modification of `maxHeaderListSize(int value)`